squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +131,39 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    await this.server.updatePlayerList;
+    let admins = await this.server.getAdminsWithPermission('canseeadminchat');
+
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (admins.includes(player.steamID)) {
```

Comment:
For cleaner code, it might be better to negate this and then use `continue`.

```js
for (const player of this.server.players) {
  if (!admins.includes(player.steamID)) continue;

  this.amountAdmins++;
  await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
}
````